packages/core/src/components/chip/chip.tsx

```diff
@@ -57,13 +57,26 @@ export class TdsChip {
     checked?: boolean;
   }>;
 
+  /** @internal Emit checked value if type is radio */
+  @Event({
+    eventName: 'internalRadioOnChange',
+    composed: true,
+    cancelable: false,
+    bubbles: true,
+  })
+  internalRadioOnChange: EventEmitter<{
+    chipId: string;
+    checked: boolean;
+  }>;
```

Comment:
```suggestion
  internalRadioOnChange: EventEmitter<{
    chipId: string;
    checked: boolean;
    chipGroup: string;
  }>;
```

We need to send in a `name` prop too as it serves as group identifier. If we do not have it, reset of checked state will reset all radio Chip on the page, no matter they do not belong to the same group.